src/box/lua/upgrade.lua

```diff
@@ -1408,6 +1420,7 @@ local handlers = {
     {version = mkversion(2, 10, 4), func = upgrade_to_2_10_4},
     {version = mkversion(2, 10, 5), func = upgrade_to_2_10_5},
     {version = mkversion(2, 11, 0), func = upgrade_to_2_11_0},
+    {version = mkversion(2, 11, 1), func = upgrade_to_2_11_1},
```

Comment:
Please add a downgrade handler and a test as well.

Reply:
The absence of `max_id` is handled correctly but is not tested, so I wrote downgrade handler and test.